lib/game.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require_relative 'player.rb'
+
+# class tic-tac-toe game
+class Game
+  attr_accessor :board, :end_game
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @player1 = Player.new('Player 1', 'X')
+    @player2 = Player.new('Player 2', 'O')
+    @current_player = @player1
+    @end_game = false
+  end
+
+  def draw_board
+    puts "\n"
```

Comment:
`main.rb` file should be responsible for user interaction. This is the only one place where puts and get.chomp can be used.